docs/sources/writing-guide/tooling-and-workflows/run-a-local-webserver/index.md

```diff
@@ -57,6 +57,17 @@ To specifically build Grafana and Grafana Cloud documentation:
 make docs PROJECTS='grafana grafana-cloud'
 ```
 
+Let's say that you have forked the main project repository, so your local working directory name doesn't match the project name.
+You can use the `PROJECTS` option to define the local cloned repository (see the Arguments section below).
+
+```bash
+make docs PROJECTS="tempo::tempo-doc-work"
+```
+
+The format is `<PROJECT>[:VERSION[:REPOSITORY[:DIR]]].`
+The example mounts the `PROJECT` tempo, at the default `VERSION` latest, using the `REPOSITORY` `tempo-doc-work`, and the default `DIR` `docs/sources`.
+This example builds the Tempo documentation from the local working directory, `tempo-doc-work` instead of the standard `tempo` directory.
```

Comment:
```suggestion
This example builds the Tempo documentation from the local working directory, `tempo-doc-work`, instead of the standard `tempo` directory.
```